Mathlib/LinearAlgebra/ExteriorPower/Basic.lean

```diff
@@ -234,4 +234,52 @@ theorem map_comp (f : M →ₗ[R] N) (g : N →ₗ[R] N') :
     map n (g ∘ₗ f) = map n g ∘ₗ map n f := by
   aesop
 
+/-! Linear equivalences in degrees 0 and 1. -/
+
+variable (R M) in
+/-- The linear equivalence ` ⋀[R]^0 M ≃ₗ[R] R`. -/
+@[simps! (config := .lemmasOnly) apply symm_apply]
+noncomputable def linearEquiv₀ : ⋀[R]^0 M ≃ₗ[R] R :=
```

Comment:
Perhaps just `zeroEquiv` to match  `PiTensorProduct.subsingletonEquiv`?

Reply:
I have made the change.